app/models/manageiq/providers/redhat/infra_manager/event_parser.rb

```diff
@@ -59,7 +59,7 @@ def self.ems_ref_from_object_in_event(data)
     ManageIQ::Providers::Redhat::InfraManager.make_ems_ref(data[:href])
   end
 
-  def self.parse_new_target(full_data, message, ems)
+  def self.parse_new_target(full_data, message, ems, name)
```

Comment:
Can you name this `event_type` instead of `name`?  Its not clear what it is the name of